js/bitmax.js

```diff
@@ -491,7 +737,7 @@ module.exports = class bitmax extends Exchange {
             request['from'] = request['to'] - limit * duration * 1000 - 1;
```

Comment:
https://bitmax.io/api/pro/v1/barhist?symbol=BTC%2FUSDT&interval=1&from=1588291200000&to=1588291800001

↑ This looks like a legit URL, however, it returns an empty result and this is not documented in the exchange's API docs. The API doc does not mention that you can't go far in the past with `from` + `to`. Unfortunately, having `from` + `to` + `n` is very non-intuitive.

Reply:
P.S. it should be either `from` + `n` or `from` + `to`, but not all three.